btree/btree.c

```diff
@@ -1067,6 +1067,9 @@ struct node_type {
 	/** Captures node data in segment */
 	void (*nt_capture)(struct slot *slot, struct m0_be_tx *tx);
 
+	/** Returns the header size for credit calculation of tree operations */
+	int  (*nt_credit_size)(void);
```

Comment:
The name should be indicating the operation. In this case it could be nt_create_delete_credit_size.
This is because the same will be used for other credit calculations on this node too like record_add and record_del !!!

Reply:
Updated.